OpenRA.Game/Network/ReplayConnection.cs

```diff
@@ -36,15 +36,18 @@ class Chunk
 		public readonly bool IsValid;
 		public readonly Session LobbyInfo;
 		public readonly string Filename;
+		public bool IsOldReplay;
 
 		public ReplayConnection(string replayFilename)
 		{
 			Filename = replayFilename;
+			var replayMetadata = ReplayMetadata.Read(replayFilename);
 
 			// Parse replay data into a struct that can be fed to the game in chunks
 			// to avoid issues with all immediate orders being resolved on the first tick.
 			using (var rs = File.OpenRead(replayFilename))
 			{
+				IsOldReplay = replayMetadata.GameInfo.FinalGameTick == 0;
```

Comment:
Replays aren't backwards compatible (all the other engine changes lead to immediate desyncs), so there is no need for this.